src/scripts/lint/.eslintrc.react.js

```diff
@@ -31,6 +31,8 @@ module.exports = {
     '@typescript-eslint/no-empty-function': ['error', { allow: ['arrowFunctions'] }],
     'react/jsx-filename-extension': ['error', { extensions: ['.tsx'] }],
     'react/jsx-props-no-spreading': 'off',
+    // Promotes localization of string literals in JSX
+    'react/jsx-no-literals': ['error', { noStrings: false }],
```

Comment:
Shouldn't this rule be part of our app? I don't see this being honored in other projects

Reply:
I don't know the scope of this project and it's goals. Folks were talking about adding Storybook composition with SLDS for React this morning to this project. That seems rather specific, too.

Reply:
I thought we were supposed to add as many lint rules as possible over here. Who else is using this?

Reply:
We are not supposed to be writing any strings `{'Hello'}` or `Hello`.